dependencies/HaXml-1.25.3/src/Text/XML/HaXml/Wrappers.hs

```diff
@@ -31,7 +31,7 @@ fix2Args :: IO (String,String)
 fix2Args = do
   args <- getArgs
   when ("--version" `elem` args) $ do
-      putStrLn $ "part of HaXml-"++ VERSION
+      putStrLn $ "part of HaXml-"
```

Comment:
These changes allow `stack repl` to work, because `stack` had a problem with `VERSION`